recipe/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -ex
+
+pushd streamlit_antd_components/frontend
+npm install antd@5.15.1 --save-exact
```

Comment:
Umm, why is that necessary? The package asks for 5.7.2 (https://github.com/nicedouble/StreamlitAntdComponents/blob/c2d59692add9d5f87124f78f046ce8f20444e718/streamlit_antd_components/frontend/package.json#L9). Did they somehow forget to push the package.json?

And this makes me wonder if the commit we picked is the right one. I would suggest that we loko at the diff between the wheel and the commit and see what differs and see which commit matches the wheel the most?